config/packages/cache.yaml

```diff
@@ -15,5 +15,7 @@ framework:
         #app: cache.adapter.apcu
 
         # Namespaced pools use the above "app" backend by default
-        #pools:
+        pools:
             #my.dedicated.cache: null
+          cache.filesystem:
+            adapter: cache.adapter.filesystem
```

Comment:
I'm not sure this is necessary? It looks like you're autowiring `CacheInterface`, so that would use the `cache.app` pool and whatever adapter it's using (which I think is filesystem)

Reply:
That is very true. 

I wanted to be explicit that I used the filesystem. I guess it is a matter of taste.